### PR TITLE
Malaria Cell Image Classification

### PR DESCRIPTION
**Pull request by sitammeur**
# Pull Request for DL-Simplified 💡

## Issue Title :  Malaria Cell Image Classification

- **Info about the related issue (Aim of the project)** :  Goal of this project is to explore deep learning models that can accurately predict whether a cell image is parasitized or uninfected with malaria.
- **Name:**  Sitam Meur
- **GitHub ID:**  sitamgithub-MSIT
- **Email ID:**  sitammeur@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)**  Codepeak Participant

Closes: #393

### Describe the add-ons or changes you've made 📃

Solution is implemented with technologies like Keras, Keras-cv and tensorflow. Also, it has the latest keras3 support.
Used tensorflow dataset and split the data into test and training datasets.
Performed the necessary pre-processing, image resizing, rescaling, and augmentation.
Created a learning rate scheduler.
Trained models like VGG-19, Resnet-50, and EfficientNetB0 using transfer learning method.
Checked the validation accuracy and loss as graph representation.
At the end, accuracy of all models were compared by line plot, heat map and distribution plot.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

The changes that I have made for that I thoroughly tested in my kaggle kernel as well collab . I also added proper requirements in requirements.txt file for reproducebility in other machines too. Also model graphs are also added and accuracy table of all models are added the proves the working of the code.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — APPROVED** [review]
Probably the best project I have seen during this open source event. Very well documented, clean codes, really happy to see your project.

Your PR is approved and ready to be merged.
@sitamgithub-MSIT
